### PR TITLE
call to action btn and text

### PR DESCRIPTION
**Pull request by nicocapalbo**

### CODE REVIEW

File: pfdp/homepage.html [hunk 1]
```diff
@@ -98,7 +102,10 @@ <h1 class="white">Title here</h1>
     <!-- BLOCK CTA - ✅ GO -->
 
     <div class="wrapper bg-white">
-      <div class="container-bm">cta</div>
+      <div class="container-bm">
+        <p class='cta-text'>Alors, prêt à vous lancer dans cette aventure solidaire ?</p>
```
Comment (cecile-dzy-ncl) [inline pfdp/homepage.html]: this should be a h3

File: pfdp/homepage.html [hunk 2]
```diff
@@ -98,7 +102,10 @@ <h1 class="white">Title here</h1>
...
+        <p class='btn-primary'>Créer ma page de collecte</p>
```
Comment (cecile-dzy-ncl) [inline pfdp/homepage.html]: there is a btn class already existing in the global css files ... 
i always use btn-bm that i adapt to my needs + the bg-nameoftheprimarycolor for the background color

File: pfdp/css/style.css [hunk 3]
```diff
@@ -1,3 +1,5 @@
 @import "../css/variables.css";
 @import "../css/fonts.css";
 @import "../css/utilities.css";
+@import "../css/cta.css";
```
Comment (cecile-dzy-ncl) [inline pfdp/css/style.css]: buttons.css

File: pfdp/css/cta.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
+.cta-text {
```
Comment (cecile-dzy-ncl) [inline pfdp/css/cta.css]: this should be a h3 so put it in the font.css file

Comment (cecile-dzy-ncl) [reply pfdp/css/cta.css]: and without the class, just style the h3 tag

File: pfdp/css/cta.css [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+  font-weight: 600;
+  font-size: 32px;
+  line-height: 39px;
+  text-align: center;
+}
+
+.btn-primary {
```
Comment (cecile-dzy-ncl) [inline pfdp/css/cta.css]: name of the btn class should be btn-bm, check what's already existing in the global css file and adapt to your need

File: pfdp/css/cta.css [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+  color: white;
+  background-color: var(--red);
+  height: 80px;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/cta.css]: never put height and width (but do put some padding)

File: pfdp/css/cta.css [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+  width: 407px;
+  text-align: center;
+  border-radius: 53px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: 33%;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/cta.css]: 😱 
if you want to center a block or inlin-block element, you do it the following way :
`margin: 0 auto;`
here the margin is already in button global css

**Review by cecile-dzy-ncl — CHANGES_REQUESTED** [review]

File: pfdp/css/button.css [hunk 8]
```diff
@@ -0,0 +1,17 @@
+.btn-bm {
+  color: white;
+  padding: 30px;
+  border-radius: 53px;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  margin: 30px 25%;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/button.css]: nooooooooooooo 
tsss
`margin: 0 auto`

File: pfdp/css/button.css [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+}
+
+@media screen and (max-width: 500px) {
+  
+  .btn-bm {
+    margin: 10px 0%;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/button.css]: same here

**Review by cecile-dzy-ncl — CHANGES_REQUESTED** [review]

File: pfdp/css/button.css (hunk 9 above)
Comment (nicocapalbo) [reply pfdp/css/button.css]: removed

File: pfdp/css/button.css (hunk 8 above)
Comment (nicocapalbo) [reply pfdp/css/button.css]: removed margin, inserted **width: fit-content;**

File: pfdp/css/button.css [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+  width: fit-content;
+}
+
+@media screen and (max-width: 500px) {
+  
+  .btn-bm {
+    padding: 15px;
```
Comment (cecile-dzy-ncl) [inline pfdp/css/button.css:14]: why the change of padding in responsive ?

**Review by cecile-dzy-ncl — CHANGES_REQUESTED** [review]

**Pull request merged**